index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Sam - Portofolio</title>
+    <title>Sam Portofolio home page</title>
```

Comment:
Kindly note that the title seems a bit too long, Sam's Portfolio would be a good title here 🤗